tests/validate-urls.rb

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'English'
+require 'json'
+diff = `git diff --name-only --diff-filter=AM origin/master...HEAD entries/`.split("\n")
```

Comment:
```suggestion

@status = 0
diff = `git diff --name-only --diff-filter=AM origin/master...HEAD entries/`.split("\n")
```

Reply:
The build will fail regardless of the exit code since `::error::` is outputted.
Do you still want the exit code to be set?

Reply:
I think it's good style to use the return code of the script to indicate success or failure, and not rely on the caller to parse the output, especially since the output goes to `stdout` and not `stderr` here.